src/modules/output/ansi_output.rs

```diff
@@ -14,25 +14,23 @@ pub struct AnsiOutput {
 
 impl IStdout for AnsiOutput {
    fn write_str(&self, string: &str) -> io::Result<usize> {
-       let out = &self.handle;
-       let mut handle = out.lock();
-        write!(handle, "{}", string)?;
-       handle.flush();
-        Ok(0)
+        let out = &self.handle;
```

Comment:
For some reason there are 5 spaces instead of 4 now.

Reply:
no, the line 16 has only 3 spaces so it looks like 5.

Reply:
Thanks for reviewing! I'll cargo fmt and all spaces and tabs will be right ;)

Reply:
Sorry about that, the original function was missing one space of indentation, so I tried to fix it, but I think I missed line 16. My bad.

Reply:
Don't worry already fixed it! But thanks for your pull request!